desktop/core/src/desktop/js/reactComponents/MetricsComponent/MetricsComponent.tsx

```diff
@@ -0,0 +1,149 @@
+// to you under the Apache License, Version 2.0 (the
+// 'License'); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an 'AS IS' BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import React, { useState, useEffect, useRef } from 'react';
+import MetricsTable, { MetricsResponse } from './MetricsTable';
+import { Spin, Input, Select, Alert } from 'antd';
+import { get } from 'api/utils';
+import { SearchOutlined } from '@ant-design/icons';
+import './MetricsComponent.scss';
+
+const { Option } = Select;
+
+const MetricsComponent: React.FC = (): JSX.Element => {
+  const [metrics, setMetrics] = useState<MetricsResponse>();
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState<string>();
+  const [searchQuery, setSearchQuery] = useState<string>('');
+  const [selectedMetric, setSelectedMetric] = useState<string>('');
+  const [showAllTables, setShowAllTables] = useState(true);
+  const [filteredMetricsData, setFilteredMetricsData] = useState<MetricsData[]>([]);
+  const dropdownRef = useRef(null);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const response = await get<MetricsResponse>('/desktop/metrics/', { format: 'json' });
+        setMetrics(response);
+      } catch (error) {
+        setError(error.message);
+      } finally {
+        setLoading(false);
+      }
+    };
+
+    fetchData();
+  }, []);
+
+  useEffect(() => {
+    if (!metrics) {
+      return;
+    }
+
+    const filteredData = parseMetricsData(metrics).filter(tableData =>
+      tableData.dataSource.some(data => data.name.toLowerCase().includes(searchQuery.toLowerCase()))
+    );
+
+    setFilteredMetricsData(filteredData);
+  }, [searchQuery, metrics]);
+
+  const parseMetricsData = (data: MetricsResponse) => {
+    return Object.keys(data.metric)
+      .filter(
+        key =>
+          !key.startsWith('auth') &&
+          !key.startsWith('multiprocessing') &&
+          !key.startsWith('python.gc')
+      )
+      .map(key => ({
+        caption: key,
+        dataSource: Object.keys(data.metric[key]).map(subKey => ({
+          name: subKey,
+          value: data.metric[key][subKey]
+        }))
+      }));
+  };
+  const handleMetricChange = (value: string) => {
+    setSelectedMetric(value);
+    setShowAllTables(value === '');
+  };
+
+  const handleFilterInputChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setSearchQuery(e.target.value);
+  };
+
+  return (
+    <div className="cuix antd metrics-component">
+      <Spin spinning={loading}>
+        {!error && (
+          <>
+            <Input
```

Comment:
I have removed the box shadow from the input filter metrics ( ! important).
Earlier it showed a blue outline as soon as u click the input filter.